x-pack/plugins/canvas/public/apps/workpad/routes.js

```diff
@@ -28,7 +29,11 @@ export const routes = [
             dispatch(resetAssets());
             router.redirectTo('loadWorkpad', { id: newWorkpad.id, page: 1 });
           } catch (err) {
-            notify.error(err, { title: `Couldn't create workpad` });
+            notify.error(err, {
+              title: i18n.translate('xpack.canvas.apps.workpad.createWorkpadErrorMessage', {
```

Comment:
@maryia-lapata how do you think if we need to use .apps namespace?

Reply:
delete apps